packages/neuron-wallet/src/controllers/wallets.ts

```diff
@@ -123,21 +117,18 @@ class WalletsController {
   public static importKeystore = ({
     name,
     password,
-    keystore,
+    key,
```

Comment:
Why renaming this? Doesn't `keystore` look more suitable than `key`?

Reply:
get it